moped-database/README_seed_data.md

```diff
@@ -0,0 +1,39 @@
+### How to generate new seeds
+
+1. Navigate to `./moped-database`
+2. Start the hasura cluster
+3. Add / edit whatever data
+4. ❗️Warning: adding subprojects creates a circular reference that will cause seeds to fail
+5. With pg_dump installed, run this command in your shell. You can add additional tables as needed - do not include lookup tables - these tables are hydrated through migrations:
```

Comment:
You may want to consider not requiring the install of pg_dump on the machine. You could maybe do something like `docker exec -it moped-database_moped-pgsql_1 <pg_dump command here>` or similar to use the one that is installed and from the same version as the docker state database?

Should be good either way, and this is super helpful!

Reply:
i was hoping you'd chime in with a docker example! using the already-running docker container is brilliant! 💯

Reply:
i updated the readme:

```shell
# grab postgres image name from currently running container
$ PG_IMAGE=$(docker ps -f name='moped-database_hasura_1' |  grep 'moped-database_hasura_1'| awk '{ print $2 }')

# run pg_dump using postgres image
$ docker run -it --rm --network host \
  -v "$(pwd)"/seeds:/seeds $PG_IMAGE \
  pg_dump --column-inserts --data-only --schema public \
    -t moped_proj_components \
    -t moped_proj_components_subcomponents \
    -t moped_proj_contract \
    -t moped_proj_entities \
    -t moped_proj_features \
    -t moped_proj_financials \
    -t moped_proj_fiscal_years \
    -t moped_proj_funding \
    -t moped_proj_milestones \
    -t moped_proj_notes \
    -t moped_proj_partners \
    -t moped_proj_personnel \
    -t moped_proj_personnel_roles \
    -t moped_proj_phases \
    -t moped_proj_tags \
    -t moped_project \
    -t moped_project_files \
    -t moped_user_followed_projects \
    -t moped_users \
    postgres://moped:moped@localhost:5432/moped > seeds/1602292389297_initial_seed_staging.sql
```

Reply:
btw, I couldn't get `docker exec` to work b/c we need our local dir mounted to the container